tests/test_external_account_authorized_user.py

```diff
@@ -87,27 +89,35 @@ def test_default_state(self):
         assert not creds.token
         assert not creds.valid
         assert not creds.requires_scopes
+        assert not creds.scopes
+        assert creds.token_info_url
+        assert creds.client_id
+        assert creds.client_secret
         assert creds.is_user
+        assert creds.refresh_token == REFRESH_TOKEN
+        assert creds.audience == AUDIENCE
+        assert creds.token_url == TOKEN_URL
```

Comment:
Can you add a test for the new logic in cred initialization?

Reply:
Done